presto-tests/src/test/java/com/facebook/presto/execution/TestQueues.java

```diff
@@ -140,7 +140,7 @@ public void testResourceGroupManager()
         waitForQueryState(queryRunner, secondDashboardQuery, RUNNING);
     }
 
-    @Test(timeOut = 60_000)
+    @Test(timeOut = 120_000)
```

Comment:
Is there a reason for the test to take double the time?

Reply:
This original timeout was 24_000 when this test was introduced ([pr](https://github.com/prestodb/presto/pull/9105/files#diff-13092bdc8fb4f08c4271e85ed36dd9d7342ddb3284dc8235964e7ddd975f1f37R111)) and recently it was changed to 60_000 before fixing this flaky test ([pr](https://github.com/prestodb/presto/commit/8ebf7d99061399ea11ae94d0bd2254911e142687)).  It timed out in ci ([example](https://github.com/prestodb/presto/pull/16738/checks?check_run_id=3667324332)) as well as in my local after multiple run, so increasing it. This test is basically creating multiple queries and waiting for different status, so depending on the instance load during the test run time, this test can take up time.

Reply:
fyi disabled this test as it was flaky. Make sure it's not flaky before enabling again. https://github.com/prestodb/presto/pull/16793